### ISSUE TITLE
Train-Test Split for Celeb-DF

### ISSUE THREAD

**Issue by ucalyptus2** [niyunsheng/CORE#3]
@niyunsheng Wonderful project, how do u train-test split for Celeb-DF?

**Comment by niyunsheng** [comment]
There is a `List_of_testing_videos.txt` file in the official Celeb-DF dataset. 
This file shows which videos belong to the test set and others belong to the training set
